tests/utils.py

```diff
@@ -5,14 +5,18 @@
 import itertools
 from asyncio import Event, Lock
 
-import asynctest
 from hypothesis.internal.reflection import (
     define_function_signature,
     impersonate
 )
 
 from server.db import FAFDatabase
 
+from .exhaust_callbacks import exhaust_callbacks
+
+# Imported for re-export
+_ = exhaust_callbacks
```

Comment:
Should probably define an __all__

Reply:
Doesn't it make more sense to import this directly from its file everywhere instead of importing it here and doing nothing with it?

Reply:
It is actually used within the file, but I think it makes sense to keep the util functions being imported form the same file (as much as I hate 'util' modules)